docs/pages/app.md

```diff
@@ -10,7 +10,13 @@ You can integrate Sketch into your app or workflow using the `sketch://` URL sch
 
 **Note:** If a document is already open the canvas view does not change.
 
-## Center layer and specify zoom level
+### Table of Contents
+
+- [Center layer and specify zoom level](#centerlayer)
+- [Run a plugin command](#runplugincommand)
+- [Add a Remote Library](#addremotelibrary)
```

Comment:
Looking at the main Sketch documentation, maybe using the term shared Library is better overall

```suggestion
- [Add a shared Library](#add-shared-library)
```

Reply:
Changed.